tendermint/src/error.rs

```diff
@@ -40,6 +40,58 @@ pub enum Kind {
     /// Signature invalid
     #[error("bad signature")]
     SignatureInvalid,
+
+    /// invalid Vote type
+    #[error("invalid Type")]
+    InvalidMessageType,
+
+    /// Negative block height
+    #[error("negative height")]
+    NegativeHeight,
+
+    /// Negative voting round
+    #[error("negative round")]
+    NegativeRound,
+
+    /// Negative POL round
+    #[error("negative pol round")]
+    NegativePOLRound,
```

Comment:
```suggestion
    NegativePolRound,
```

Convention in Rust is to use strict CamelCase (I know, it looks weird…)

Reply:
Covered in the next commit.